src/main/java/io/managed/services/test/client/kafkamgmt/KafkaMgmtApiUtils.java

```diff
@@ -75,11 +75,11 @@ public static Optional<KafkaRequest> getKafkaByOwner(KafkaMgmtApi api, String ow
     }
 
     public static KafkaRequestPayload defaultKafkaInstance(String name) {
-        return new KafkaRequestPayload()
-            .name(name)
-            // TODO enterprise provide .clusterId(Environment.CLUSTER_ID) and .marketplace(Environment.IS_ENTERPRISE ? "enterprise" : null) with new SDK
```

Comment:
what to do with this?

Reply:
This are fields which are supposed to be added when creating kafka instance,  we were not able to provide them in older version of SDK, so if you could pass these @rkpattnaik780  (the values as visible in comment are supposed to be read from `Environment` class, which serves to read values which are passed as properties/ environment variables, which is also the way how we will provide values of these fields).